src/components/MultiSelect/MultiSelect-story.js

```diff
@@ -20,15 +18,29 @@ import {
 import { withReadme } from 'storybook-readme';
 import readme from './README.md';
 import MultiSelect from '../MultiSelect';
+import { componentsX } from '../../internal/FeatureFlags';
```

Comment:
Just one remainder of `v10` feature flag ref.

Reply:
removed, thanks!